### PR TITLE
fixed absence instument translation for MSVC building

### PR DESCRIPTION
**Pull request by handrok**
add lrelease for MSVC building

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -778,11 +778,23 @@ add_custom_target(lupdate
 
 ## lrelease is used to compile the translated text files generated by lupdate into binary resources
 
-add_custom_target(lrelease
-   COMMAND Qt5::lrelease ${PROJECT_SOURCE_DIR}/share/locale/*.ts
-   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+if (MSVC)
```
Comment (Jojo-Schmitz) [inline CMakeLists.txt]: I'd drop the ` if`

File: CMakeLists.txt [hunk 2]
```diff
@@ -778,11 +778,23 @@ add_custom_target(lupdate
...
+   file(GLOB TS_FILES
+    "share/locale/*.ts"
    )
 
+   add_custom_target(lrelease
+      COMMAND Qt5::lrelease 
+      ${TS_FILES}
+      WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+   )
+else (MSVC)
```
Comment (Jojo-Schmitz) [inline CMakeLists.txt]: and the entire else branch

File: CMakeLists.txt [hunk 3]
```diff
@@ -778,11 +778,23 @@ add_custom_target(lupdate
 
 ## lrelease is used to compile the translated text files generated by lupdate into binary resources
 
-add_custom_target(lrelease
-   COMMAND Qt5::lrelease ${PROJECT_SOURCE_DIR}/share/locale/*.ts
-   WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
+if (MSVC)
+   file(GLOB TS_FILES
+    "share/locale/*.ts"
```
Comment (Jojo-Schmitz) [inline CMakeLists.txt]: adding `${PROJECT_SOURCE_DIR}/` here

Comment (Jojo-Schmitz) [reply CMakeLists.txt]: unfortunately that doesn't seem to work, so leave this to the COMMAND part

**Comment by anatoly-os** [comment]
Merged manually https://github.com/musescore/MuseScore/commit/9ccc6cc86134d44095b102af14ddee02a2a123cb. @Jojo-Schmitz's notes were taken into account.

**Pull request closed without merging**